client/disputes/new-evidence/cover-letter.tsx

```diff
@@ -13,39 +14,455 @@ import {
+				attachments.push(
+					`• ${ __(
+						'Customer signature',
+						'woocommerce-payments'
+					) } (${ __(
+						'Attachment',
+						'woocommerce-payments'
+					) } ${ String.fromCharCode( 64 + attachmentCount ) })`
+				);
+			}
+			if ( dispute.evidence?.refund_policy ) {
+				attachmentCount++;
+				attachments.push(
+					`• ${ __(
+						'Store refund policy',
+						'woocommerce-payments'
+					) } (${ __(
+						'Attachment',
+						'woocommerce-payments'
+					) } ${ String.fromCharCode( 64 + attachmentCount ) })`
+				);
+			}
+			if ( dispute.evidence?.shipping_documentation ) {
+				attachmentCount++;
+				attachments.push(
+					`• ${ __(
+						'Proof of shipping',
+						'woocommerce-payments'
+					) } (${ __(
+						'Attachment',
+						'woocommerce-payments'
+					) } ${ String.fromCharCode( 64 + attachmentCount ) })`
+				);
+			}
+			if ( dispute.evidence?.service_documentation ) {
+				attachmentCount++;
+				attachments.push(
+					`• ${ __(
+						'Service documentation',
+						'woocommerce-payments'
+					) } (${ __(
+						'Attachment',
+						'woocommerce-payments'
+					) } ${ String.fromCharCode( 64 + attachmentCount ) })`
+				);
+			}
+			if ( dispute.evidence?.cancellation_policy ) {
+				attachmentCount++;
+				attachments.push(
+					`• ${ __(
+						'Cancellation policy',
+						'woocommerce-payments'
+					) } (${ __(
+						'Attachment',
+						'woocommerce-payments'
+					) } ${ String.fromCharCode( 64 + attachmentCount ) })`
+				);
+			}
+			if ( dispute.evidence?.access_activity_log ) {
+				attachmentCount++;
+				attachments.push(
+					`• ${ __(
+						'Access activity log',
+						'woocommerce-payments'
+					) } (${ __(
+						'Attachment',
+						'woocommerce-payments'
+					) } ${ String.fromCharCode( 64 + attachmentCount ) })`
+				);
+			}
+			if ( dispute.evidence?.uncategorized_file ) {
+				attachmentCount++;
+				attachments.push(
+					`• ${ __(
+						'Additional documentation',
+						'woocommerce-payments'
+					) } (${ __(
+						'Attachment',
+						'woocommerce-payments'
+					) } ${ String.fromCharCode( 64 + attachmentCount ) })`
+				);
+			}
+
+			// If no attachments were provided, use default list
+			const attachmentsList =
+				attachments.length > 0
+					? attachments.join( '\n' )
+					: `• ${ __(
+							'AVS/CVV Match: Billing address and security code matched',
+							'woocommerce-payments'
+					  ) } (${ __( 'Attachment', 'woocommerce-payments' ) } A)
+• ${ __(
+							'IP/Device Data: Location and device info used at purchase',
+							'woocommerce-payments'
+					  ) } (${ __( 'Attachment', 'woocommerce-payments' ) } B)
+• ${ __(
+							'Customer Confirmation: Email or chat confirming purchase',
+							'woocommerce-payments'
+					  ) } (${ __( 'Attachment', 'woocommerce-payments' ) } C)
+• ${ __(
+							'Usage Data: Login records for the digital goods',
+							'woocommerce-payments'
+					  ) } (${ __( 'Attachment', 'woocommerce-payments' ) } D)`;
```

Comment:
It could be an issue with an encoding for the print view, but the bullets do not look good in it:

<img width="713" alt="image" src="https://github.com/user-attachments/assets/43d7b7e5-00dd-42df-84c9-8670a9608a6f" />

Reply:
oh shoot, that's bad. fixed it here 2d5df615fed2bff47408020a384943aea9bbe037. thanks for the call out